tests/mic/test_audio_input.py

```diff
@@ -0,0 +1,25 @@
+"""
+Mock tests for microphone input
+"""
+from unittest import mock
+
+from spokestack.mic.pyaudio import PyAudioMicrophoneInput
+from spokestack.config import SpeechConfig
+
+
+def test_audio_input():
+
+    params = {"sample_rate": 16000, "frame_width": 10}
+    config = SpeechConfig(params)
+    mic = PyAudioMicrophoneInput(config)
+    mic.audio = mock.MagicMock()
+    mic.build()
+    # start audio stream
+    mic.start()
+    assert mic.is_active()
+    # read a single frame
+    frame = mic.read()
+    assert frame
```

Comment:
it would be good to verify that you get the expected number of samples from this call